engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/SpelExpression.scala

```diff
@@ -140,43 +139,25 @@ class SpelExpression(parsed: ParsedSpelExpression,
   }
 }
 
-class SpelExpressionParser(expressionFunctions: Map[String, Method],
+class SpelExpressionParser(// for SpelExpression creation
+                           expressionFunctions: Map[String, Method],
                            expressionImports: List[String],
                            classLoader: ClassLoader,
+                           propertyAccessors: Seq[PropertyAccessor],
```

Comment:
why do we extract it to constructor? Will we need it in the future?

Reply:
It is used in `SpelExpression` in creation of `EvaluationContext`. My main point in this change was to move factory logic from `SpelExpressionParser` to factory method (`default`), because then we can easily modify `SpelExpressionParser` (by some with... method) without recreating it from scratch. We could think about also moving factory logic from `SpelExpression.evaluate` and then we will replace these few properties with one `createEvaluationContext: () => EvaluationContext`. What do you think about it?